src/cli/cli.cpp

```diff
@@ -1961,6 +1961,7 @@ void Interpreter::ProcessRouterSelectionJitter(int argc, char *argv[])
     else
     {
         SuccessOrExit(error = ParseLong(argv[0], value));
+        VerifyOrExit(0 < value && value < 256, error = kThreadError_InvalidArgs);
```

Comment:
The spec actually defines the default value for ROUTER_SELECTION_JITTER to be exactly 120 seconds.  While the Thread Spec calls this a "default" value, it does not specify any bounds.